custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -211,6 +211,8 @@ outer_predict <- function(input_filename, target_type, model=NULL, positive_clas
 
     if (!isFALSE(read_input_data_hook)) {
         data <- read_input_data_hook(input_filename)
+    } else if (grepl(".mtx", input_filename, fixed=TRUE)) {
```

Comment:
I would always check for "endswith()" instead of "includes()"

```
grepl("\\.mtx$", input_filename)
```

Reply:
we dont want "hello.mtxfoo" or "hello.mtx.txt" match

Reply:
good catch, I'll change this.